source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -49,14 +64,24 @@ class ProcessorState : public Logger::Loggable<Logger::Id::filter> {
   mutableBody(envoy::service::ext_proc::v3alpha::ProcessingRequest& request) const PURE;
 
 protected:
+  Filter* const parent_;
```

Comment:
const ref

I think we've also started moving towards using a more descriptive name than parent, so maybe `filter_`?

Reply:
Ok!